django/contrib/admin/templates/admin/widgets/related_widget_wrapper.html

```diff
@@ -21,10 +23,17 @@
         {% if can_delete_related %}
         <a class="related-widget-wrapper-link delete-related" id="delete_id_{{ name }}"
             data-href-template="{{ delete_related_template_url }}?{{ url_params }}"
+            data-popup="yes"
             title="{% blocktranslate %}Delete selected {{ model }}{% endblocktranslate %}">
             <img src="{% static 'admin/img/icon-deletelink.svg' %}" alt="{% translate 'Delete' %}">
         </a>
         {% endif %}
+        {% if can_view_related %}
+        <a class="related-widget-wrapper-link view-related" id="view_id_{{ name }}"
+            data-href-template="{{ change_related_template_url }}">
+            <strong></strong>
+        </a>
```

Comment:
Excellent patch! I want this so bad :D

I think this could look really nice if, instead of rendering a text link, we used the existing "view" icon, e.g.:

```
<img src="{% static 'admin/img/icon-viewlink.svg' %}" alt="{% translate 'View' %}">
```

Looks like this:

<img width="133" alt="Screen Shot 2022-02-10 at 11 28 42 AM" src="https://user-images.githubusercontent.com/126461/153452058-f82d4737-94cd-4ee2-aa15-f413e68b4f24.png">